diff --git a/dest/node/benchmark/timer.js b/dest/node/benchmark/timer.js
index 96fda5aea2396b3a649b8b4b00d10f3953e39dbe..b51582a46e4209ee03761a15c47b313e277c3d35 100644
--- a/dest/node/benchmark/timer.js
+++ b/dest/node/benchmark/timer.js
@@ -12,7 +12,7 @@
  */
 export class Timer {
     constructor() {
-        this.start = performance.now();
+        this.start = Date.now();
     }
     us() {
         return this.ms() * 1000;
@@ -25,7 +25,7 @@ export class Timer {
      * @returns The elapsed time in milliseconds.
      */
     ms() {
-        return performance.now() - this.start;
+        return Date.now() - this.start;
     }
     /**
      * Returns the time elapsed since the Timer instance was created, in seconds.
diff --git a/dest/node-cjs/benchmark/timer.js b/dest/node-cjs/benchmark/timer.js
index 3c4ad2c1931abda2d798117f637ab6a46b6bc811..43ea7b901efe60b8e9174f55e61ad41f9f5ff61f 100644
--- a/dest/node-cjs/benchmark/timer.js
+++ b/dest/node-cjs/benchmark/timer.js
@@ -15,7 +15,7 @@ exports.Timer = void 0;
  */
 class Timer {
     constructor() {
-        this.start = performance.now();
+        this.start = Date.now();
     }
     us() {
         return this.ms() * 1000;
@@ -28,7 +28,7 @@ class Timer {
      * @returns The elapsed time in milliseconds.
      */
     ms() {
-        return performance.now() - this.start;
+        return Date.now() - this.start;
     }
     /**
      * Returns the time elapsed since the Timer instance was created, in seconds.
diff --git a/src/benchmark/timer.ts b/src/benchmark/timer.ts
index 81dfe37f127711ffeffa9ec0250e314140c5e684..3727d1fe23fe773bf017dc2e058ed7a189f90080 100644
--- a/src/benchmark/timer.ts
+++ b/src/benchmark/timer.ts
@@ -14,7 +14,7 @@ export class Timer {
   private start: number;
 
   constructor() {
-    this.start = performance.now();
+    this.start = Date.now();
   }
 
   public us() {
@@ -29,7 +29,7 @@ export class Timer {
    * @returns The elapsed time in milliseconds.
    */
   public ms() {
-    return performance.now() - this.start;
+    return Date.now() - this.start;
   }
 
   /**
