diff --git a/node_modules/@aztec/foundation/dest/timer/timer.js b/node_modules/@aztec/foundation/dest/timer/timer.js
index 253ca99..7069b7e 100644
--- a/node_modules/@aztec/foundation/dest/timer/timer.js
+++ b/node_modules/@aztec/foundation/dest/timer/timer.js
@@ -12,7 +12,8 @@
  */
 export class Timer {
     constructor() {
-        this.start = performance.now();
+        // this.start = performance.now();
+        this.start = Date.now();
     }
     /**
      * Return microseconds.
@@ -28,7 +29,8 @@ export class Timer {
      * @returns The elapsed time in milliseconds.
      */
     ms() {
-        return performance.now() - this.start;
+        // return performance.now() - this.start;
+        return Date.now() - this.start;
     }
     /**
      * Returns the time elapsed since the Timer instance was created, in seconds.